includes/admin/views/html-admin-page-status-report.php

```diff
@@ -84,55 +81,33 @@
 			</td>
 		</tr>
 		<tr>
-			<td data-export-label="WP Version"><?php esc_html_e( 'WordPress version', 'woocommerce' ); ?>:</td>
-			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of WordPress installed on your site.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+			<td data-export-label="CMS Version"><?php esc_html_e( 'CMS version', 'woocommerce' ); ?>:</td>
+			<td class="help"><?php echo wc_help_tip( esc_html__( 'The CMS version installed on your site.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
 			<td>
 				<?php
-				$latest_version = get_transient( 'woocommerce_system_status_wp_version_check' );
-
-				if ( false === $latest_version ) {
-					$version_check = wp_remote_get( 'https://api.wordpress.org/core/version-check/1.7/' );
-					$api_response  = json_decode( wp_remote_retrieve_body( $version_check ), true );
-
-					if ( $api_response && isset( $api_response['offers'], $api_response['offers'][0], $api_response['offers'][0]['version'] ) ) {
-						$latest_version = $api_response['offers'][0]['version'];
-					} else {
-						$latest_version = $environment['wp_version'];
-					}
-					set_transient( 'woocommerce_system_status_wp_version_check', $latest_version, DAY_IN_SECONDS );
-				}
-
-				if ( version_compare( $environment['wp_version'], $latest_version, '<' ) ) {
-					/* Translators: %1$s: Current version, %2$s: New version */
-					echo '<mark class="error"><span class="dashicons dashicons-warning"></span> ' . sprintf( esc_html__( '%1$s - There is a newer version of WordPress available (%2$s)', 'woocommerce' ), esc_html( $environment['wp_version'] ), esc_html( $latest_version ) ) . '</mark>';
+				if ( function_exists( 'classicpress_version' ) ) {
+				    /* translators: ClassicPress version string */
+				    printf( __( 'You are running ClassicPress version %s', 'woocommerce' ), classicpress_version() );
```

Comment:
`printf()` will not have `%s` link value to add. Consider using this for both options
```
echo '<mark>' . sprintf( esc_html__( 'You are running ClassicPress version - %1$s', 'woocommerce' ), esc_html( classicpress_version() )  ) . '</mark>';
```

Reply:
This is incorrect, try it. There is no difference between `printf` and `echo sprintf`.

If you wanted to add the `<mark>` tag that would be a reason to use `sprintf`. However I wouldn't change it because the original code didn't have this.

Reply:
`echo '<mark class="error"><span class="dashicons dashicons-warning"></span>` existed before. 

We need to use the translatable version of the line thus the `%1$`. How would you suggest we do this?

Reply:
This code came from my suggestion. It will translate correctly as written.

Reply:
You're right about the other part though, the wrapping tag should be restored. In this case it's fine to write it all on one line with `sprontf` like it was previously, or split across multiple lines and use `printf`.

When there is only one placeholder in a translatable string, `%s` is the correct syntax. `%1$s` would probably work fine too butthe `1` marker is not needed because there's only one placeholder.

Reply:
I don't see that wrapping tags are needed since it is just displaying the output with no formatting to indicate status (ie whether up-to-date or not). 
![version](https://user-images.githubusercontent.com/46998578/66263159-b5a85a80-e839-11e9-9902-db85e054b7da.png)

Reply:
The fix I have pushed helps to sort this out as in image below.

![Screen Shot 2019-10-20 at 10 52 12](https://user-images.githubusercontent.com/7713923/67156558-bc020080-f328-11e9-8b60-3244efb6c3f0.png)

First, it checks if the CMS is ClassicPress then runs CP stuff else does what was before in WP. So the WP compatibility is still possible.

Better ways to improve the code is to make a function that accepts an API url to make the code DRY.

Reply:
There is a little issue of the CP API that returns the latest version of stable CP release. Little research going on in the slack channels.

Reply:
Fixed API call.

As seen from the notice in the top, CP core calls for an update as well as the Plugin suggests the same thing. Similar to WP.

![Screen Shot 2019-10-20 at 18 16 23](https://user-images.githubusercontent.com/7713923/67161745-1f5e5380-f366-11e9-977d-cf184bc1bf22.png)